charts/timed/templates/service-backend.yaml

```
@@ -9,7 +9,7 @@ spec:
   type: {{ .Values.backend.service.type }}
   ports:
     - port: {{ .Values.backend.service.externalPort }}
-      targetPort: http
+      targetPort: {{ .Values.backend.service.internalPort }}
```

this might be a better fit for deployment-backend (so you can use name based ports from svc -> pod)

---

so, it should work using `http` as before:

```suggestion
      targetPort: http
```